conf/messages

```diff
@@ -2,4 +2,5 @@ error.postcode.invalid = Please enter a valid postcode
 error.utr.invalid = Please enter a valid UTR
 error.telephone.invalid = Please enter a valid telephone number
 error.no.ampersand = Agency name cannot contain "&"
-
+error.des.text.invalid = This field is limited to alphanumeric characters (A-Z, a-z, 0-9) and the following characters \\-,.)/
+error.maxLength = Maximum amount of characters exceeded
```

Comment:
Do we need this custom error.maxLength message? The (default message)[https://github.com/playframework/playframework/blob/2.5.12/framework/src/play/src/main/resources/messages.default#L12] has the advantage that it tells you what the maximum length is.